react.js

```diff
@@ -1,5 +1,47 @@
-module.exports = {
-  presets: [
-    require('babel-preset-react'),
-  ],
+module.exports = function shopifyReactPreset(context, options) {
+  options = options || {};
+
+  // eslint-disable-next-line no-process-env
+  var env = process.env.BABEL_ENV || process.env.NODE_ENV;
+
+  var plugins = [
+    // Make JSX spread operator use Object.assign instead of the Babel helper
+    [require.resolve('babel-plugin-transform-react-jsx'), {
```

Comment:
Shoot, I thought `Object.assign` was supported natively in all our supported browsers. Will add a note to the docs.